deployments/cert_manager.go

```diff
@@ -297,7 +270,7 @@ func (cm CertManager) apply(ctx context.Context, c *kubernetes.Cluster, ui *term
 		"spec" : {
 			"isCA"       : true,
 			"commonName" : "epinio-ca",
-			"secretName" : "%s",
+			"secretName" : "epinio-ca-root",
```

Comment:
I would (still) prefer to have this in the `rootCAName` constant, placed at the beginning, easy to find.

Reply:
Done: https://github.com/epinio/epinio/pull/1009/commits/d71e84f0a2943e7ffb579e79a70a38090b549cd2